test/MapTest.php

```diff
@@ -1,37 +1,29 @@
-<?php
+<?php declare(strict_types=1);
 
 namespace Amp\ParallelFunctions\Test;
 
-use Amp\MultiReasonException;
+use Amp\CompositeException;
 use Amp\PHPUnit\AsyncTestCase;
 use function Amp\ParallelFunctions\parallelMap;
-use function Amp\Promise\wait;
 
 class MapTest extends AsyncTestCase
 {
     public function testValidInput()
     {
-        $this->assertSame([3, 4, 5], wait(parallelMap([1, 2, 3], function ($input) {
+        $r = parallelMap([1, 2, 3], function ($input) {
             return $input + 2;
-        })));
-    }
-
-    public function testCorrectOutputOrder()
-    {
-        $this->assertSame([0, 1, 0], wait(parallelMap([0, 1, 0], function ($input) {
-            \sleep($input);
-
-            return $input;
-        })));
+        });
+        ksort($r);
```

Comment:
@trowski Too bad we no longer keep the order.

Reply:
I guess we could sort again based on the original array.

Reply:
Shall I implement this?

Reply:
Done!